using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md

```diff
@@ -1,24 +1,31 @@
-- Resource idle by cluster: A resource-specific (CPU, RAM, etc.) breakdown view of Infra Idle for all of your clusters and nodes.
+- **Idle by type**: A high-level view of your wasted spend for each cluster broken down by Workload Idle and Infra Idle.
+- **Resource idle by workload**: A resource-specific (CPU, RAM, etc.) breakdown view of Workload Idle and Workload Efficiency for all of your Kubernetes workloads.
+- **Resource idle by cluster**: A resource-specific (CPU, RAM, etc.) breakdown view of Infra Idle for all of your clusters and nodes.
 
 Let’s establish some definitions and then explore each of these views.
 ## Definitions
-- **Workload Idle**- Workload Idle is defined as the cost of resources which are requested, but not used, by Kubernetes workloads.
-For example, in a pod with a single container which requests 2 Gi of memory but only uses 1 Gi, there is 1 Gi of workload idle which is assigned a cost. This is a new definition of ‘idle’ for Kubecost.
-- **Infra Idle** - Infra Idle (Infrastructure Idle) is defined as the difference between the cost of allocated resources and the cost of the hardware on which they run. Allocation is defined as the max of usage and request.
-For example, a Kubernetes node which has 32 Gi of usable memory runs two pods each with one container. The first pod requests 2 Gi of memory but only uses 1 Gi. The second pod specifies no requests but uses 3 Gi. There is 27 Gi of Infra Idle which is assigned a cost. This is the same definition of ‘idle’ as used in the Kubecost Allocations page (see more details here).
-- **Total Idle** - The sum of Infra Idle and Workload Idle. 
-- **Cluster Efficiency** - The ratio of the cost of used resources to the total cost of all resources. (cost of resources used) / (total cost of resources). This is the efficiency metric presented in the upper right-hand corner of the Overview page. 
+- _**Workload Idle**_- Workload Idle is defined as the cost of resources which are requested, but not used, by Kubernetes workloads.
+For example, in a pod with a single container which requests 2 Gi of memory but only uses 1 Gi, there is 1 Gi of workload idle which is assigned a cost. **This is a new definition of ‘idle’ for Kubecost.**
+- _**Infra Idle**_ - Infra Idle (Infrastructure Idle) is defined as the difference between the cost of allocated resources and the cost of the hardware on which they run. Allocation is defined as the max of usage and request.
+For example, a Kubernetes node which has 32 Gi of usable memory runs two pods each with one container. The first pod requests 2 Gi of memory but only uses 1 Gi. The second pod specifies no requests but uses 3 Gi. There is 27 Gi of Infra Idle which is assigned a cost. This is the same definition of ‘idle’ as used in the Kubecost Allocations page (see more details [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/cost-allocation/efficiency-idle#idle)).
```

Comment:
When you reference other locations in the docs, don't use a static link like this, use a relative link. Reason being is so links don't become brittle and makes the content portable.